autode/values.py

```diff
@@ -0,0 +1,401 @@
+import numpy as np
+from autode.log import logger
+from abc import ABC, abstractmethod
+from typing import Union
+from copy import deepcopy
+from autode.units import (Unit,
+                          ha, kjmol, kcalmol, ev,
+                          ang, a0, nm, pm, m,
+                          rad, deg,
+                          ha_per_ang, ha_per_a0, ev_per_ang)
+
+
+def _to(value,
+        units: Union[Unit, str]):
+    """Convert a value or value array to a new unit and return a copy
+
+    Arguments:
+        units (autode.units.Unit | str):
+
+    Returns:
+        (autode.values.Value):
+    """
+    if value.units == units:
```

Comment:
does the account for units being a string? I think it does - just checking though

Reply:
Yep :ok_hand: